INSTALL.md

```diff
@@ -183,3 +183,10 @@ $ cmake ... \
 ```
 
 ⚠️ CompilerGym requires exactly LLVM 10.0.0.
+
+# Contribution
```

Comment:
This isn't necessary, as `make init` installs the pre-commit hooks:

https://github.com/facebookresearch/CompilerGym/blob/34712e9ec3bd9326bc9f26e025b8c97b74cef8c6/Makefile#L152-L154